runbotics/runbotics-orchestrator-ui/src/views/utils/routerUtils.ts

```diff
@@ -0,0 +1,11 @@
+import { NextRouter } from "next/router";
+
+export const ReplaceQueryParams = (queryParams: Record<string, any>, router: NextRouter): void => {
```

Comment:
or maybe lets do the hook for it so we don't need to pass router obj every time - then return single fn that starts with lower case

( ͡° ͜ʖ ͡°)

Reply:
Yeah! Sounds cool, I'il do that